kolibri/core/assets/src/views/tabs/index.vue

```diff
@@ -1,76 +1,33 @@
 <template>
 
-  <ui-tabs
-    ref="tabs"
-    class="tabs"
-    :type="type"
-    :disableRipple="true">
-    <ui-tab
-      v-for="item in items"
-      :title="item.title"
-      :icon="item.icon"
-      :selected="itemIsSelected(item)"
-      :disabled="itemIsDisabled(item)"/>
-  </ui-tabs>
+  <nav class="tabs">
+    <ul>
+      <slot name="items"/>
+    </ul>
+  </nav>
 
 </template>
 
 
 <script>
 
-  module.exports = {
-    props: {
-      // Array that contains objects consisting of title, icon, selected, disabled
-      items: {
-        type: Array,
-      },
-      type: {
-        type: String,
-        validator(type) {
-          const validTypes = ['text', 'icon', 'icon-and-text'];
-          return validTypes.includes(type);
-        },
-        default: 'icon-and-text',
-      },
-    },
-    components: {
-      'ui-tabs': require('keen-ui/src/UiTabs'),
-      'ui-tab': require('keen-ui/src/UiTab'),
-    },
-    methods: {
-      addTabListeners() {
-        const tabs = Array.from(this.$refs.tabs.$el.querySelectorAll('.ui-tab-header-item'));
-        tabs.forEach((tab, index) => {
-          tab.addEventListener('click', () => {
-            this.$emit('tabclicked', index);
-          }, false);
-        });
-      },
-      itemIsSelected(item) {
-        return item.selected && (item.selected === true);
-      },
-      itemIsDisabled(item) {
-        return item.disabled && (item.disabled === true);
-      },
-    },
-    mounted() {
-      this.$nextTick(this.addTabListeners);
-    },
-    beforeDestroy() {
-      // TODO: Remove event listeners
-    },
-  };
+  module.exports = {};
 
 </script>
 
 
-<style lang="stylus" scoped></style>
+<style lang="stylus" scoped>
 
+  @require '~kolibri.styles.definitions'
 
-<style lang="stylus">
+  .tabs
+    background-color: white
+    white-space: nowrap
+    overflow-x: auto
+    overflow-y: hidden
```

Comment:
Adding overflow-x hidden to `.tab-title` (to prevent text from overflowing in case the text is too long) results in the need for this. 

![overflow](https://cloud.githubusercontent.com/assets/7193975/25975383/d91204a2-3662-11e7-9fc4-8434e2b95a04.gif)